framework/src/problems/DisplacedProblem.C

```diff
@@ -289,10 +284,7 @@ DisplacedProblem::updateMesh(bool mesh_changing)
   if (haveFV())
   {
     _mesh.markFiniteVolumeInfoDirty();
-    _mesh.buildFiniteVolumeInfo();
-    _mesh.computeFiniteVolumeCoords();
-    _mesh.cacheFaceInfoVariableOwnership();
-    _mesh.cacheFVElementalDoFs();
+    _mesh.setupFiniteVolumeMeshData();
```

Comment:
Is it necessary to call both `markFiniteVolumeInfoDirty` and `setupFiniteVolumeMeshData`? Will the latter call early return if the info is not dirty?

Reply:
No, you are right there is no early return anymore. Will fix this.